pocld/snakeoil.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
What is the purpose of the key/cert ?

Reply:
I assumed he had a test that uses these, but seems not.

Reply:
Oh, yeah those were lying around from some unrelated stuff and weren't supposed to get picked up